fake_switches/arista/arista_core.py

```diff
@@ -19,6 +19,7 @@
 from fake_switches.arista.command_processor.config import ConfigCommandProcessor
 from fake_switches.arista.command_processor.config_vlan import ConfigVlanCommandProcessor
 from fake_switches.arista.command_processor.default import DefaultCommandProcessor
+from fake_switches.arista.command_processor.terminal_display import TerminalDisplay
```

Comment:
I think it's confusing if TerminalDisplay is a command processor or not regardless of imports.
Suggestion: `fake_switches.arista.command_processor.display.terminal`